pkg/helper/versions/helpers.go

```diff
@@ -32,13 +32,23 @@ func GetVersionList(r *rosa.Runtime, channelGroup string, isSTS bool, isHostedCP
 	}
 
 	for _, v := range vs {
+		if isHostedCP {
+			if v.HostedControlPlaneDefault() {
+				defaultVersion = v.RawID()
+			}
+
+		} else {
+			if v.Default() {
+				defaultVersion = v.RawID()
+			}
+		}
```

Comment:
```
		if (isHostedCP && v.HostedControlPlaneDefault()) || v.Default() {
				defaultVersion = v.RawID()
		}
```

Minior nit. Is this logically equivalent to the if block above?

Reply:
Yep I moved it to a function